libcxx/test/std/algorithms/robust_re_difference_type.compile.pass.cpp

```diff
@@ -240,9 +240,14 @@ TEST_CONSTEXPR_CXX20 bool all_the_algorithms()
     (void)std::sort(first, last, std::less<void*>());
     (void)std::sort_heap(first, last);
     (void)std::sort_heap(first, last, std::less<void*>());
-    if (!TEST_IS_CONSTANT_EVALUATED) (void)std::stable_partition(first, last, UnaryTrue());
-    if (!TEST_IS_CONSTANT_EVALUATED) (void)std::stable_sort(first, last);
-    if (!TEST_IS_CONSTANT_EVALUATED) (void)std::stable_sort(first, last, std::less<void*>());
+#if TEST_STD_VER < 26
+    if (!TEST_IS_CONSTANT_EVALUATED)
+#endif
+    {
+      (void)std::stable_partition(first, last, UnaryTrue());
+      (void)std::stable_sort(first, last);
+      (void)std::stable_sort(first, last, std::less<void*>());
```

Comment:
Why did you change `stable_sort`, and why is the CI happy?

Reply:
This part was drive-by. I changed the lines because they were adjacent to the line for `stable_partition`. Previously, constexpr `stable_sort` was implemented by #110320, so the CI is happy.

Reply:
I saw an open `stable_sort` PR, but I overlooked that is the `ranges`  version. My bad.